src/models/itemspage.cpp

```diff
@@ -200,3 +200,13 @@ bool ItemsPage::contains(const QString &id) const
     std::tie(i, j) = findItem(id);
     return i != -1;
 }
+
+QStringList ItemsPage::allArrangedItems() const
+{
+    QStringList result;
+    for (const QStringList & pageItems : qAsConst(m_pages)) {
```

Comment:
```suggestion
    for (const QStringList & pageItems : std::as_const(m_pages)) {
```

qAsConst 被废弃了，应该也会有编译警告。

Reply:
done，代码中有两处使用，已都修改为std::as_const